### PR TITLE
fix linting errors

### PR DESCRIPTION
**Pull request by benwaffle**
~The only linting errors left are 'line longer than 100 chars'. Do we care about this type of error?~ fixed

### CODE REVIEW

**Comment by ctumolosus** [comment]
Yes :)

File: .eslintrc [hunk 1]
```diff
@@ -4,5 +4,8 @@
     "node": true,
     "mocha": true,
     "jest": true
+  },
+  "rules": {
+    "function-paren-newline": 0
```
Comment (ctumolosus) [inline .eslintrc]: Opinionated 😅

File: src/generators/node_graphql/utilities/GraphQLSchemaConfig.js [hunk 2]
```diff
@@ -42,50 +48,53 @@ class GraphQLQuery {
...
 
-    // get a single instance of this resource
+    // Get a single instance of this resource
     } else if (!isEnclosingType(resultType)
             && resultType.fullyQualifiedType.fullyQualifiedType.match(strOrVersion(this.config.resource.type))) {
 
       if (this.config.isPrimaryGetter) {
         return camelCase(this.config.resource.type.shortName);
-      } else {
+      } else { // eslint-disable-line no-else-return
```
Comment (ctumolosus) [inline src/generators/node_graphql/utilities/GraphQLSchemaConfig.js]: Should respect rules 😅

File: src/generators/node_graphql/utilities/GraphQLSchemaConfig.js [hunk 3]
```diff
@@ -42,50 +48,53 @@ class GraphQLQuery {
...
   }
 
   get name() {
-    const { path, resultType, } = this.config.operation;
+    const { path, resultType } = this.config.operation;
     const staticParts = path.split('/').filter(x => x.length > 0 && x[0] !== ':');
     const queryParts = path.split('/').filter(x => x.length > 0 && x[0] === ':');
 
-    // get multiple instances of this resource
+    /* eslint-disable padded-blocks */
```
Comment (ctumolosus) [inline src/generators/node_graphql/utilities/GraphQLSchemaConfig.js]: Should respect rules 😅

File: .eslintrc (hunk 1 above)
Comment (benwaffle) [reply .eslintrc]: I want to be able to write things like

```js
      this.config.responses.find(
        flow(
          get('code.integer.value'),
          inRange(200, 300),
        ),
      ),
```

File: src/generators/node_graphql/utilities/GraphQLSchemaConfig.js (hunk 2 above)
Comment (benwaffle) [reply src/generators/node_graphql/utilities/GraphQLSchemaConfig.js]: I think it looks better if I don't follow this rule...

**Comment by benwaffle** [comment]
All of the linter errors are fixed. I'll fix the tests in a separate PR

**Pull request merged**